app/src/lib/score.ts

```diff
@@ -1,17 +1,70 @@
-import { CurrentGameSubscription } from "generated/graphql"
+import { CurrentGameSubscription, GameStatsQuery } from "generated/graphql"
 import { Team } from "lib/team"
-import { filter, flatMap } from "lodash"
+import { filter, flatten, sum } from "lodash"
 
 export function teamScore(
   team: Team,
-  turns: CurrentGameSubscription["games"][0]["turns"],
+  turns: GameStatsQuery["turns"],
   players: CurrentGameSubscription["games"][0]["players"]
 ) {
   const teamPlayerIds = filter(players, (player) => player.team === team).map(
     (player) => player.id
   )
-  const teamTurns = filter(turns, (turn) =>
+  const completedTurns = filter(turns, (turn) => !!turn.review_started_at)
+  const teamTurns = filter(completedTurns, (turn) =>
     teamPlayerIds.includes(turn.player_id)
   )
-  return flatMap(teamTurns, (turn) => turn.completed_card_ids).length
```

Comment:
Remove dependency on completed_card_ids for this calculation since each scoring record per turn has a "score" (0, -1, 1, etc.) that will be used to determine score (no longer just number of cards)

We'll keep completed_card_ids around as a convenience for other use cases like "which cards are left in the bowl?" -- but we could also use the scoring records that are not status "skipped"